pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/filter/postfilter/osteo/OldOsteoActivityCode.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsm.model.filter.postfilter.osteo;
+
+import java.util.Arrays;
+
+public enum OldOsteoActivityCode {
+
+    ABOUTCHILD,
+    PARENTAL_CONSENT,
+    RELEASE_MINOR,
+    RELEASE_SELF,
+    PREQUAL,
+    ABOUT_YOU,
+    CONSENT,
+    LOVEDONE,
+    CONSENT_ASSENT;
+
+    public static String[] asStrings() {
```

Comment:
I think returning array of Strings kinda breaks the idea of enums, as I see these values are used in steam for boolean value, I would create method something like `isOldOsteoActivityCode` and I would use `valueOf` method in it wrapped by `try/catch` and would return boolean value.